ddtrace/contrib/asgi/middleware.py

```diff
@@ -130,13 +132,28 @@ async def __call__(self, scope, receive, send):
         if sample_rate is not None:
             span.set_tag(ANALYTICS_SAMPLE_RATE_KEY, sample_rate)
 
+        host_header = None
+        for key, value in scope["headers"]:
+            if key == b"host":
+                try:
+                    host_header = value.decode("ascii")
+                except UnicodeDecodeError:
+                    log.warning(
+                        "failed to decode host header, url will contain the hostname of the sever", exc_info=True
```

Comment:
```suggestion
                        "failed to decode host header, url will contain the hostname of the server", exc_info=True
```

also "url will contain the hostname of the server" is a little ambiguous, can't we be more direct with something like "Host from http headers will not be considered"